### DISCUSSION TITLE
Trying to figure best hwaccel_args without trying them all

### DISCUSSION THREAD

**Discussion by The-NinthWave** [blakeblackshear/frigate#5343] [Ask A Question] [answered]
I have an NVIDIA Quadro P600 that I would like to use for h/w acceleration.

Here is an output:
```
ffprobe -rtsp_transport tcp rtsp://nick:secret@10.0.40.10:554/Streaming/Channels/1

Input #0, rtsp, from 'rtsp://nick:secret@10.0.40.15:554/Streaming/Channels/1':
  Metadata:
    title           : Media Presentation
  Duration: N/A, start: 0.066000, bitrate: N/A
  Stream #0:0: Video: h264 (Main), yuvj420p(pc, progressive), 2688x1520 [SAR 1:1 DAR 168:95], 15 fps, 15 tbr, 90k tbn
```
Since the stream is h264, I would use "h264_cuvid" but in a thread, I have seen a guy using "h264_v4l2m2m" which is not even shown on https://docs.frigate.video/configuration/hardware_acceleration

So I am a bit confused

Any explanation / recommendation ?

While on this question, I have this cheaper camera's output:
```
Input #0, rtsp, from 'rtsp://10.0.40.11:554/user=admin_password=secret_channel=1_stream=0.sdp':
  Metadata:
    title           : RTSP Session
  Duration: N/A, start: 0.066667, bitrate: N/A
  Stream #0:0: Video: h264 (Main), yuvj420p(pc, bt709, progressive), 704x480, 15 fps, 15 tbr, 90k tbn

```
Any override at the camera level or same global parameters would fit ?

**Comment by NickM-27** [comment, marked as answer]
Not sure what you mean, `h264_v4l2m2m ` is in those docs. It is for the raspberry pi. You should use the hwaccel recommended for nvidia in the nvidia section. 

> Any override at the camera level or same global parameters would fit ?

It's h264 so it would be the same

**Reply by The-NinthWave** [reply]
> Not sure what you mean, `h264_v4l2m2m ` is in those docs. It is for the raspberry pi. You should use the hwaccel recommended for nvidia in the nvidia section.
> 
> > Any override at the camera level or same global parameters would fit ?
> 
> It's h264 so it would be the same

Again, you save me from my blindness.  When I found the thread about this HIKvisions camera, I did not look what the guy's host was so I just assumed it was linux and so I looked under the related subsection.

Thanks

**Comment by The-NinthWave** [comment]
In fact, the guy has a Win10 Docker.... but still
I am running Debian on a XEON so "h264_cuvid" it is.

**Reply by NickM-27** [reply]
You need to have an nvidia GPU. If you don't then that is incorrect

**Reply by The-NinthWave** [reply]
> You need to have an nvidia GPU. If you don't then that is incorrect

Yes, as stated in original post, this is for an NVIDIA Quadro
